src/Template/Element/sentences_lists/sentence_in_list.ctp

```diff
@@ -0,0 +1,43 @@
+<?php
+if (!isset($sentenceId)) {
+    $sentenceId = 'sentence.id';
+}
+if (!isset($ngRepeat)) {
+    $ngRepeat = '';
+} else {
+    $ngRepeat = 'ng-repeat="'.$ngRepeat.'"';
+}
+if (!isset($canRemove)) {
+    $canRemove = false;
+}
+$editableCss = $canRemove ? 'removeable' : '';
+?>
+
+<div class="sentence-and-translations-wrapper <?= $editableCss ?>" layout="row" layout-align="center start" 
+     ng-class="{'removed': vm.isRemoved[<?= $sentenceId ?>]}"
+     <?= $ngRepeat ?>>
+
+    <div flex class="removed-confirmation md-whiteframe-1dp" ng-if="vm.isRemoved[<?= $sentenceId ?>]">
+        <?= __('Sentence removed from list.') ?>
+    </div>
+
+    <?= $this->element('sentences/sentence_and_translations', $sentenceAndTranslationsParams); ?>
+
+    <?php if ($canRemove) { ?>
+    <div class="remove-from-list" ng-if="!vm.isRemoved[<?= $sentenceId ?>]">
+        <md-button class="md-icon-button md-warn" ng-click="vm.removeSentence(<?= $sentenceId ?>)">
+            <md-icon>remove_circle_outline
+                <md-tooltip><?= __('Remove from list') ?></md-tooltip>
+            </md-icon>
+        </md-button>
+    </div>
+
+    <div class="remove-from-list" ng-if="vm.isRemoved[<?= $sentenceId ?>]">
+        <md-button class="md-icon-button" ng-click="vm.undoRemoval(<?= $sentenceId ?>)">
+            <md-icon>undo
+                <md-tooltip><?= __('Undo') ?></md-tooltip>
```

Comment:
About the tooltip "Undo", maybe it’s just me, but I find it not clear enough, because it lacks context.

![undo](https://user-images.githubusercontent.com/5107734/75980988-813ae680-5f1e-11ea-81c0-21a548e8f6db.png)

I asked myself "undo what"?
I suggest you either put the button next to the text "Sentence removed from list", or make the tooltip say "Undo removal" or something clearer.